micromamba/tests/test_create.py

```diff
@@ -1167,6 +1171,25 @@ def test_create_check_dirs(tmp_home, tmp_root_prefix):
         assert os.path.isdir(env_prefix / "lib" / "python3.8" / "site-packages" / "traitlets")
 
 
+@pytest.mark.parametrize("shared_pkgs_dirs", [True], indirect=True)
+def test_create_python_site_packages_path(tmp_home, tmp_root_prefix):
+    env_name = "myenv"
+    env_prefix = tmp_root_prefix / "envs" / env_name
+    # imagesize is a noarch: python package
+    cmd = ["-n", env_name, "python=3.13", "python-freethreading", "imagesize=1.4.1"]
+    helpers.create(*cmd)
+
+    assert os.path.isdir(env_prefix)
+
+    if platform.system() == "Windows":
+        assert os.path.isdir(env_prefix / "lib" / "site-packages" / "imagesize")
```

Comment:
So Windows stores all packages together indifferently from the version of Python and now from the free threaded builds?

Can explicit tests be added for `python=3.13t` for Windows for the presence or absence of folders?